x-pack/plugin/eql/qa/common/src/main/resources/test_queries_unsupported.toml

```diff
@@ -224,18 +224,6 @@ sequence
 '''
 expected_event_ids  = [1, 2, 2, 3]
 
-
-[[queries]]
-query = '''
-sequence with maxspan=1d
-  [file where event_subtype_full == "file_create_event"] by file_path
-  [process where opcode == 1] by process_path
-  [process where opcode == 2] by process_path
-  [file where event_subtype_full == "file_delete_event"] by file_path
-| head 4
-| tail 2'''
-expected_event_ids  = [67, 68, 69, 70, 72, 73, 74, 75]
-
 [[queries]]
 query = '''
 sequence with maxspan=1h
```

Comment:
@rw-access this test and the rest should not return any results; any `maxspan` lower than 15h actually.
Entries 67 and 68 have the same timestamp (131509374395921780) but 69 has a timestamp of 131509374446778110 meaning the distance between the beginning of the sequence and the 69 is:
131509374446778110 - 131509374395921780 = 50856330 millis or ~14h.

Am I missing something?

Reply:
The timestamp units are windows filetime. One unit is 1e-7 seconds. I'm guessing that's what you're running into

Reply:
Thanks. This would need to be aligned with ECS/unix timestamp.